db_structure.sql

```diff
@@ -18,11 +18,32 @@ CREATE TABLE Brands (
     name VARCHAR(255)
 );
 
+CREATE TYPE gender AS ENUM ('1', '2', '3');
+CREATE TYPE goals AS ENUM ('1', '2', '3', '4', '5');
+CREATE TYPE skin_types AS ENUM ('1', '2', '3', '4', '5');
+CREATE TYPE skin_concerns AS ENUM ('1', '2', '3', '4', '5');
+CREATE TYPE skin_conditions AS ENUM ('1', '2', '3', '4', '5');
+CREATE TYPE routine_products AS ENUM ('1', '2', '3', '4', '5', '6', '7', '8', '9');
+CREATE TYPE products_used AS ENUM ('1', '2', '3', '4', '5', '6', '7', '8', '9', '10', '11', '12', '13', '14');
+CREATE TYPE skincare_brands AS ENUM ('1', '2', '3', '4', '5', '6', '7', '8', '9', '10', '11', '12', '13', '14', '15');
+CREATE TYPE makeup_brands AS ENUM ('1', '2', '3', '4', '5', '6', '7', '8', '9', '10', '11', '12');
+
 CREATE TABLE Users (
     id SERIAL PRIMARY KEY,
     email VARCHAR(100) NOT NULL UNIQUE,
     password VARCHAR(256) NOT NULL,
-    full_name VARCHAR(256) NOT NULL
+    full_name VARCHAR(256) NOT NULL,
+    gender gender,
+    birthdate DATE,
+    goals goals,
+    skin_types skin_types,
+    skin_concerns skin_concerns,
+    skin_conditions skin_conditions,
+    routine_products routine_products,
+    makeup BOOLEAN,
```

Comment:
Not sure what this means? If it's a boolean, it should have a name that makes us understand what is going on.

For example:

* using_makeup
* has_makeup
* likes_makeup

I'm not sure what this "makeup" refers to?